### PR TITLE
Export CloudFormation Templates

### PR DESCRIPTION
**Pull request by root-beer**

### CODE REVIEW

**Comment by root-beer** [comment]
Please review and merge PR. @Joeper214 🙇 
```sh
# skyhopper/frontend
$ npm i
$ gulp ts
```

**Comment by Joeper214** [comment]
I'll review now.

File: app/assets/javascripts/cf_templates.js [hunk 1]
```diff
@@ -91,8 +100,65 @@
...
+      },
+      confirm_export: function () {
+        var self = this;
+        var html = $('<div>', {class: 'panel panel-info', style: 'margin-bottom: 0px'});
+        html.append(
+          $('<div>', {class: 'panel-heading', text: t('cf_templates.msg.confirm_export')}).prepend(
+            $('<span>', {class: 'glyphicon glyphicon-info-sign'})
+          ),
+          $('<ul>').append(
+            _.map(this.selections, function (obj) {
```
Comment (Joeper214) [inline app/assets/javascripts/cf_templates.js]: I think we don't have to use lodash.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map

JavaScript (ES5) supports `filter` and `map` functions.

Comment (Joeper214) [inline app/assets/javascripts/cf_templates.js]: So that we could achieve a more faster and native JavaScript code in the App.

Comment (Joeper214) [inline app/assets/javascripts/cf_templates.js]: Another tips here:

https://github.com/cht8687/You-Dont-Need-Lodash-Underscore

Comment (root-beer) [inline app/assets/javascripts/cf_templates.js]: Thanks! I'll fix it.

**Comment by Joeper214** [comment]
LGTM Thanks! @root-beer  🙇

**Comment by root-beer** [comment]
🙇

**Pull request merged**